examples/apps/generic_client/generic_client.py

```diff
@@ -116,14 +115,203 @@ def ParseCommandLine(args):
         json.dumps(config)
     except pconfig.ConfigurationException as e:
         logger.error(str(e))
+        config = None
+
+    return config
+
+
+def _retrieve_uri_from_registry_list(config):
+    # Retrieve Http JSON RPC listener uri from registry
+    logger.info("\n Retrieve Http JSON RPC listener uri from registry \n")
+    # Get block chain type
+    blockchain_type = config['blockchain']['type']
+    if blockchain_type == "Ethereum":
+            worker_registry_list = EthereumWorkerRegistryListClientImpl(
+                config)
+    else:
+        worker_registry_list = None
+        logger.error("\n Worker registry list is currently supported only for \
+            ethereum block chain \n")
```

Comment:
This results in embedded spaces in the error message. Instead use:

`logger.error("\n Worker registry list is currently supported only for "`
`            "ethereum block chain \n")`

Reply:
ok